backend/src/db-utils.js

```diff
@@ -284,6 +301,13 @@ const queryDashboardBlocksAndTxs = async ({ dataSource }) => {
       });
     }
   });
+
+  if (dataSource === DS_INDEXER_BACKEND) {
+    blocks.forEach((block) => {
+      block.timestamp = new BN(block.timestamp).divn(10 ** 6).toNumber();
+    });
+  }
```

Comment:
We need to adjust the timestamp to be the same style as the legacy backend used to provide